src/components/PopoverButton/index.spec.tsx

```diff
@@ -131,7 +136,9 @@ describe(__filename, () => {
       throw new Error('onHideOverlay was unexpectedly empty');
     }
     // Simulate how Overlay calls onHide() internally.
-    onHideOverlay();
+    const fakeEvent = (createFakeEvent() as unknown) as React.SyntheticEvent;
```

Comment:
I had to use `unknown` here to avoid having to mock the entirety of `React.SyntheticEvent`. It seems like there should be a better solution than this though. Because we are only doing this because onHideOverlay requires an `event` based on its type (which is a type definition change introduced by v1.0.0-beta.17), I think it's ok to keep this in the test, but I'm open to suggestions.

I'm going to leave this for @willdurand to look at when he returns.